testdata/parenthesis/checker_tests.go

```diff
@@ -7,3 +7,7 @@ func badReturn() [](func()) {
 func veryBadReturn() [](func([](func()))) {
 	return nil
 }
+
+var badVar [](func())
+
+var badVar2 [5](*int)
```

Comment:
Add at least one case for `const` and `type` declarations.

Reply:
@Quasilyte, could you please show, how this check could be used in type decl?

Reply:
```go
type foo [](func())
```
For `struct` types, you need to walk all fields.